alvr/server/cpp/platform/win32/VideoEncoderVCE.cpp

```diff
@@ -180,17 +219,20 @@ amf::AMFComponentPtr VideoEncoderVCE::MakeEncoder(
 				break;
 		}
 
-		if (m_use10bit) {
-			amfEncoder->SetProperty(AMF_VIDEO_ENCODER_COLOR_BIT_DEPTH, AMF_COLOR_BIT_DEPTH_10);
-		} else {
-			amfEncoder->SetProperty(AMF_VIDEO_ENCODER_COLOR_BIT_DEPTH, AMF_COLOR_BIT_DEPTH_8);
-		}
-
 		//No noticable performance difference and should improve subjective quality by allocating more bits to smooth areas
 		amfEncoder->SetProperty(AMF_VIDEO_ENCODER_ENABLE_VBAQ, true);
 		
-		//Fixes rythmic pixelation. I-frames were overcompressed on default settings
+		//Fixes rythmic pixelation.
 		amfEncoder->SetProperty(AMF_VIDEO_ENCODER_MAX_QP, 30);
+		
+		//Turns Off IDR/I Frames
+		amfEncoder->SetProperty(AMF_VIDEO_ENCODER_IDR_PERIOD, 0);
+
+		// Disable AUD to produce the same stream format as VideoEncoderNVENC.
+		// FIXME: This option doesn't work in 22.10.3, but works in versions prior 22.5.1
+		amfEncoder->SetProperty(AMF_VIDEO_ENCODER_INSERT_AUD, false);
+
+		amfEncoder->SetProperty(AMF_VIDEO_ENCODER_VBV_BUFFER_SIZE, bitRateIn / frameRateIn);
```

Comment:
I have doubts if this is correct do to the fact that the same thing also exists on Line 405 and 410 same comment on HEVC version maybe its fine but I don't see a purpose in this existing if we already do it else where.

Reply:
I should also add that(if I remember correctly zarik also had his doubts on this being correct though that was a bit ago)

Reply:
Line 405 and 410 handle adaptive bitrate option. Since VBV buffer depends on bitrate in our formula, we probably need to update it if our requested bitrate has changed too.